.docker/Dockerfile-waf

```diff
@@ -1,73 +1,116 @@
-FROM owasp/modsecurity-crs@sha256:94c67e36f8c48553b4e51378579f6afc818a5d9d2e0c6e09cf846913ee9fac31
+ARG NGINX_VERSION="1.27.3"
+FROM nginx:${NGINX_VERSION}
 
 LABEL maintainer="USA.gov Web Ops"
 
-ARG RELEASE=3.3.2
-
 ARG GITBRANCH
-ENV GITBRANCH ${GITBRANCH:-none}
+ENV GITBRANCH=${GITBRANCH:-none}
 
```

Comment:
Interesting ... I looked this up and I see that the syntax we use elsewhere is "for backward compatibility" and "maybe removed in a future version".